Source: https://src.fedoraproject.org/rpms/rng-tools/blob/aaca5d6d9c9a50e0d0eed74c67b09863ac37149a/f/2-rt-revert-build-randstat.patch

From de2ee0d8b7e8ad2915165ef941a6ec37442a2fdc Mon Sep 17 00:00:00 2001
From: Vladis Dronov <vdronov@redhat.com>
Date: Tue, 6 Jul 2021 14:36:46 +0200
Subject: [PATCH] Revert "Build randstat binary"

We do not want new and mostly useless randstat binary.
This reverts commit 2ce93190cb0111fcab2f622a539689d70960643a.

Signed-off-by: Vladis Dronov <vdronov@redhat.com>
---
 .gitignore          | 3 +--
 contrib/Makefile.am | 5 +++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git .gitignore .gitignore
index 943fa2b..5dc334d 100644
--- .gitignore
+++ .gitignore
@@ -1,6 +1,6 @@
 *.a
 *.o
-.deps/
+/.deps/
 /aclocal.m4
 /ar-lib
 /autom4te.cache/
@@ -10,7 +10,6 @@
 /config.status
 /config.sub
 /configure
-/contrib/randstat
 /depcomp
 /install-sh
 /missing
diff --git contrib/Makefile.am contrib/Makefile.am
index a81fb0e..18c4fbc 100644
--- contrib/Makefile.am
+++ contrib/Makefile.am
@@ -1,2 +1,3 @@
-bin_PROGRAMS = randstat
-randstat_SOURCES = randstat.c
+
+EXTRA_DIST = randstat.c
+
-- 
2.26.3

